app/build.gradle.kts

```diff
@@ -50,4 +54,8 @@ dependencies {
             strictly("1.0.0-beta09")
         }
     }
+
+    androidTestImplementation("androidx.compose.ui:ui-test:1.1.0")
+    androidTestImplementation("androidx.compose.ui:ui-test-junit4:1.1.0")
+    debugImplementation("androidx.compose.ui:ui-test-manifest:1.1.0")
```

Comment:
add dependencies strings in the buildSrc

Reply:
Done